### PR TITLE
feat: Prevent overriding the value from config

### PR DESCRIPTION
**Pull request by irvifa**
## Description

Fixes #<ISSUE-NUMBER>Previously the DEBUG env var will always be overridden into True, while
we already have a way to read it from env var. By setting the default
value as True and not overridding the value later we can retainthe DEBUG
value provided from the users.

Note: It's a good idea to open an issue first for discussion.

## Checklist
- [x] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [x] **Tests** pass:   `nox -s py-3.6` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [x] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [x] Please **merge** this PR for me once it is approved.

### CODE REVIEW

**Review by glasnt — APPROVED** [review]

File: run/django/mysite/settings.py [hunk 1]
```diff
@@ -63,7 +63,7 @@
 
 # SECURITY WARNING: don't run with debug turned on in production!
 # Change this to "False" when you are ready for production
```
Comment (dandhlee) [inline run/django/mysite/settings.py:65]: Could you move this warning up above to where it's set to `True` as well?

**Review by dandhlee — CHANGES_REQUESTED** [review]
LGTM, one minor comment.

File: run/django/mysite/settings.py (hunk 1 above)
Comment (irvifa) [reply run/django/mysite/settings.py]: Addressed your comment @dandhlee

**Review by dandhlee — APPROVED** [review]
LGTM.

**Pull request merged**